skins_updater.lua

```diff
@@ -47,9 +47,8 @@ if #internal.errors > 0 then
 end
 
 -- http://minetest.fensta.bplaced.net/api/apidoku.md
-local root_url = "http://minetest.fensta.bplaced.net"
-local page_url = root_url .. "/api/v2/get.json.php?getlist&page=%i&outformat=base64" -- [1] = Page#
-local preview_url = root_url .. "/skins/1/%i.png" -- [1] = ID
+local root_url = "http://skinsdb.terraqueststudios.net"
+local page_url = root_url .. "/api/v1/content?client=mod" -- [1] = Page#
```

Comment:
This string lacks a `%i` placeholder which is used by `internal.get_pages_count` (`page_url:format(1)`). According to L122, this placeholder still exists, thus it should be moved back into this string instead of manual concat.

Reply:
done https://github.com/minetest-mods/skinsdb/pull/91/commits/e325cc5f0492246a7cde51ff39d1b6bf98f7d3ad